fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1453,7 +1453,7 @@ public class Config extends ConfigBase {
      * DO NOT use it for production env.
      */
     @ConfField
-    public static boolean allow_replica_on_same_host = false;
+    public static boolean allow_replica_on_same_host = true;
```

Comment:
this config is `"DO NOT use it for production env"`, we should not open it by default.